charm/kafka/reactive/autostart.py

```diff
@@ -27,4 +27,4 @@ def autostart_service():
             hookenv.status_set('active', 'ready')
             return
 
-    hookenv.status_set('blocked', 'failed to start kafka; check syslog')
```

Comment:
I don't see where `status` is declared. Does this need to be reverted?

Reply:
Yep!